qemu/tests/virtio_fs_share_data.py

```diff
@@ -144,6 +145,27 @@ def get_stdev(file):
         params["start_vm"] = "yes"
         env_process.preprocess(test, params, env)
 
+    # check virtiofsd info
+    cmd_get_vfsd_ver = params.get("cmd_get_vfsd_ver")
+    cmd_get_qemu_ver = params.get("cmd_get_qemu_ver")
+    if cmd_get_vfsd_ver:
+        error_context.context("Check virtiofsd info", logging.info)
+        vfsd_info = process.system_output(cmd_get_vfsd_ver, shell=True).strip()
+        pattern_ver = r'version.*(\d+.\d+.\d+)'.encode()
+        vfsd_ver = re.findall(pattern_ver, vfsd_info, re.I)[0]
+        qemu_kvm_info = process.system_output(cmd_get_qemu_ver, shell=True).strip()
+        qemu_ver = re.findall(pattern_ver, qemu_kvm_info, re.I)[0]
+        if vfsd_ver != qemu_ver:
+            test.fail("virtiofsd version is %s which is wrong,"
+                      " it should be %s" % (vfsd_ver, qemu_ver))
+        pattern_cort = r'Copyright.*\d+-(\d+)'.encode()
+        vfsd_cprt = re.findall(pattern_cort, vfsd_info)[0]
+        year_curr = process.system_output("date +%Y", shell=True).strip()
+        if int(vfsd_cprt) < int(year_curr):
+            test.fail("Copyright info is %s which is wrong,"
+                      " it should be equal or larger than %s" % (vfsd_cprt, year_curr))
+        return
```

Comment:
I suggest that we should new a case for it instead add it into case `virtio_fs_share_data`, since 
case `virtio_fs_share_data` is about testing to share data between host and guest.
For example: `virtio_fs_check_info`
CC @zhenyzha

Reply:
`virtio_fs_share_data`  contains too many types of cases, I agree with your opinion @YongxueHong